dotnet-trace/test.sh

```diff
@@ -48,14 +48,14 @@ else
    exit 1
 fi 
 
-dotnet-trace report $FILENAME topN
```

Comment:
I think it would be nice for debugging if the entire output was written to stdout.